ui/lib/core/addon/components/filter-input.ts

```diff
@@ -10,25 +10,12 @@ import { debounce, next } from '@ember/runloop';
 import type { HTMLElementEvent } from 'vault/forms';
 
 interface Args {
-  value?: string; // initial value
-  placeholder?: string; // defaults to Type to filter results
-  wait?: number; // defaults to 200
+  wait?: number; // defaults to 500
```

Comment:
500 seems like a more sensible default since it is likely that a network request will be made. I originally made this component to filter an array and 200 was a good wait time for that case which I guess is why I defaulted to that at the time.